package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "bucketclient",
   "engines": {
-    "node": ">=6.9.5"
+    "node": ">=10.16.0"
   },
-  "version": "7.2.0",
+  "version": "7.2.1",
```

Comment:
shouldn't this be 7.4.x

Reply:
@rahulreddy

Reply:
Yes, good catch!
@Katherinelaue1 Can you please change this to `7.4.5`

Reply:
Changed it, thanks @JianqinWang!